test/gerkin/features/steps/db_datasets.js

```diff
@@ -0,0 +1,65 @@
+const pactum = require('pactum');
```

Comment:
There is no `db_datasets.feature` file so please change the name of file to the `db_list.js`